timber/src/test/java/timber/log/TimberTest.kt

```diff
@@ -175,8 +175,8 @@ class TimberTest {
     }.run()
 
     assertLog()
-        .hasDebugMessage("TimberTest\$debugTreeTag", "Hello, world!")
-        .hasDebugMessage("TimberTest\$debugTreeTag", "Hello, world!")
+        .hasDebugMessage("TimberTest", "Hello, world!")
+        .hasDebugMessage("TimberTest", "Hello, world!")
```

Comment:
From the test name it looked like this is what the behavior was supposed to be, and it was failing until I changed it.